docs/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 ~~~~~~~~~
 
+0.3.0 (October 2022)
+--------------------
+
+* Update to work with pytket 1.7.
```

Comment:
```suggestion
* Update pytket version requirement to 1.7.
```
(Since it did already work with 1.7, this is probably a better description.)

Reply:
Good point, is updated!